os/android/hwcservice.cpp

```diff
@@ -298,7 +298,7 @@ HwcService::Controls::DisplayModeGetAvailableModes(uint32_t display) {
   } else {
     phyDisplay = mHwc.GetExtendedDisplay(display - 1);
   }
-  uint32_t numConfigs, configIndex;
+  uint32_t numConfigs;
```

Comment:
Suggest declaring 'numConfigs' as an 'uint8_t'  since the data range will be more than sufficient.